modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/dto/v1_0/converter/UserResourceDTOConverter.java

```diff
@@ -429,6 +468,22 @@ private AccountBrief _toAccountBrief(
 		};
 	}
 
+	private AssetLibraryBrief _toAssetLibraryBrief(Group group, User user)
+		throws PortalException {
+
+		if (group.getType() != GroupConstants.TYPE_DEPOT) {
+			return null;
+		}
+
+		return new AssetLibraryBrief() {
+			{
+				setExternalReferenceCode(group::getExternalReferenceCode);
+				setGroupId(group::getGroupId);
+				setName(() -> group.getName(user.getLocale()));
```

Comment:
no, we need the locale of the "caller of the api", not the locale of the user. I think it should be in the context